metadata.yaml

```diff
@@ -32,8 +32,9 @@ requires:
     interface: s3
     limit: 1
     description: |
-      The coordinator obtains storage info on behalf of the workers, and
-      forwards all workers the storage details over mimir-worker.
+      The coordinator obtains storage information on behalf of the workers and forwards all workers
```

Comment:
this feels like a good comment to understand the design/implementation of Mimir, but not a good public description.
I'd replace it with something like:
`interface to grant Mimir access to a s3 bucket for data storage.`

limit: 1 is self-explanatory IMHO.